README.md

```diff
@@ -33,6 +33,111 @@ Get the latest version [here](https://github.com/llorllale/cactoos-matchers/rele
 
 Java version required: 1.8+.
 
+## How to use
+
+Use our matchers inside your JUnit [`@Test`](http://junit.sourceforge.net/javadoc/org/junit/Test.html) case. We also provide `Assertion<T>` in which you can compose the behavior under test and the test case's matcher, and attempt to affirm it.
+
+We provide matchers for several different domains:
+
```

Comment:
I think the table like in https://github.com/yegor256/cactoos#our-objects-vs-their-static-methods is required before the explanation of domains.

Each domain from the table should have the reference to the corresponding section, otherwise, the users have to scroll down a lot (especially in case if they are looking for some particular matcher).
It will allow project docs to be well structured and easy for the extension.

Reply:
@dgroup I guess I should've asked what exactly was required for this task.

You want this table *instead of* what this PR provides, or *in addition to* what is provided?

Reply:
@llorllale, `in addition`.
The table before the explanation of each matcher.

Reply:
@dgroup to be clear: I didn't describe each matcher. I intend to provide just an overview of each matcher domain (package).